DTS-STN/Alpha-Site
# Small fixes for the error pages.

# Description

QA design for the error pages. Most was already done.

## Acceptance Criteria

Pages must match the Figma design.

## Test Instructions

Pages to test:
- 404.js
- 500.js
- error.js (Expired and Wrong URL)

1. navigate to /kjefhkjhf to see the 404.
2. remove a env variable to see the 500 error page.
3. subscribe with an emai, in the emails, copy paste the validation link but change the querystring to see the Wrong URL
4. For the Expired, read the code unless you want to wait 24 hours :p

## Checklist
